### PR TITLE
forms-ui-update

### PR DESCRIPTION
**Pull request by HadeerMagdy17**

### CODE REVIEW

**Comment by Ahmed-Abou-Emran** [comment]
Great job @serhj ,
One thing that we can improve is the text color of the placeholder as it is un-redable.
You can change it to a lighter color, and you can check any online tool that checks contrast ration between the foreground and background colors.

image

**Comment by Ahmed-Abou-Emran** [comment]
I also noticed, that you are using pixels for things like font-size, this is not recommended for accessibility.
I also recommend reading this article about using pixels/rems/ems and its accessibility considerations.

[surprising-truth-about-pixels-and-accessibility](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/)

**Comment by HadeerMagdy17** [comment]
done

File: src/App.css [hunk 1]
```diff
@@ -139,14 +139,14 @@ table td {
 
 @media only screen and (max-width: 576px) {
  h2 {
-    font-size: 15px !important;
+    font-size: 1.5rem !important;
```
Comment (Ahmed-Abou-Emran) [inline src/App.css]: 1 rem = default font size of the browser
by default 1rem = 16px

here you are converting 1rem => 10px

File: src/App.css [hunk 2]
```diff
@@ -61,6 +61,9 @@ form{
   border-bottom: 2px solid #fff !important;
   border-radius: 0 !important;
 }
+.custom-input::placeholder {
+  color: rgb(239, 228, 228) !important; /* Set the desired color */
```
Comment (Ahmed-Abou-Emran) [inline src/App.css]: this is good, but the placeholder text should be lighter than the text around it and it shouldn't be the same as the input text
take this example of the Facebook Login :
image
image

**Comment by Ahmed-Abou-Emran** [comment]
You should consider also adding a hover state for the buttons and links, as currently, they don't have any.

**Pull request merged**